industrial_ci/src/docker.sh

```diff
@@ -162,7 +162,8 @@ RUN echo "deb ${ROS_REPOSITORY_PATH} $UBUNTU_OS_CODE_NAME main" > /etc/apt/sourc
 RUN apt-key adv --keyserver "${APTKEY_STORE_SKS}" --recv-key "${HASHKEY_SKS}" \
     || { wget "${APTKEY_STORE_HTTPS}" -O - | sudo apt-key add -; }
 
-RUN apt-get update -qq \
+RUN sed -i "/^# deb.*multiverse/ s/^# //" /etc/apt/sources.list \
```

Comment:
Just curious, why do we need this? Does `docker` come with `multiverse` disabled?